src/tests/ftest/util/dmg_utils.py

```diff
@@ -246,50 +246,86 @@ def storage_reset(path, hosts, nvme=False, scm=False, user=None,
     return result
 
 
-def pool_create(path, host_port, scm_size, insecure=True, group=None,
-                user=None, acl_file=None, nvme_size=None, ranks=None,
-                nsvc=None, sys=None):
+# pool_create_params dictionary keys.
+INSEUCRE = "insecure"
+HOST_PORT = "host_port"
+GROUP = "group"
+USER = "user"
+ACL_FILE = "acl_file"
+NVME_SIZE = "nvme_size"
+RANKS = "ranks"
+NSVC = "nsvc"
+SYS = "sys"
+
+
+def pool_create(path, scm_size, pool_create_params=None):
     """Execute pool create command through dmg tool to servers provided.
 
     Args:
         path (str): Path to the directory of dmg binary.
-        host_port (list of str): List of Host:Port where daos_server runs.
-            Use 10001 for the default port number. This number is defined in
-            daos_avocado_test.yaml
         scm_size (str): SCM size value passed into the command.
-        insecure (bool, optional): Insecure mode. Defaults to True.
-        group (str, otional): Group with priviledges. Defaults to None.
-        user (str, optional): User with priviledges. Defaults to None.
-        acl_file (str, optional): Access Control List file path for DAOS pool.
-            Defaults to None.
-        nvme_size (str, optional): NVMe size. Defaults to None.
-        ranks (str, optional): Storage server unique identifiers (ranks) for
-            DAOS pool
-        nsvc (str, optional): Number of pool service replicas. Defaults to
-            None, in which case 1 is used by the dmg binary in default.
-        sys (str, optional): DAOS system that pool is to be a part of. Defaults
-            to None, in which case daos_server is used by the dmg binary in
-            default.
+        pool_create_params (Dictionary): Parameters passed in to dmg pool
+            create. Use the keys defined above to use non-default value.
 
     Returns:
         CmdResult: Object that contains exit status, stdout, and other
             information.
     """
     # Create and setup the command
     dmg = DmgCommand(path)
-    dmg.insecure.value = insecure
-    dmg.hostlist.value = host_port
+    # Insecure mode. Defaults to True.
+    if pool_create_params is not None and INSEUCRE in pool_create_params:
+        dmg.insecure.value = pool_create_params[INSEUCRE]
+    else:
+        dmg.insecure.value = True
+    # Comma separated list of Host:Port where daos_server runs.
+    # e.g., wolf-31:10001,wolf-32:10001. Use 10001 for the default port number.
+    # This number is defined in daos_avocado_test.yaml
+    if pool_create_params is not None and HOST_PORT in pool_create_params:
```

Comment:
You could reduce if conditions by,
if pool_create_params is not None:
    dmg.insecure.value = pool_create_params.get(INSECURE, True) // get value if key exists or return a default value.

Reply:
Thanks for suggestion. I decided to suppress the pylint error, so I'll revert to the original code.